tests/eko/io/test_struct.py

```diff
@@ -195,3 +196,20 @@ def test_load_opened(self, tmp_path: pathlib.Path, eko_factory: EKOFactory):
         read_opened = EKO.read(tmp_path, extract=False)
 
         assert read_closed.metadata == read_opened.metadata
+
+    def test_version(self, tmp_path: pathlib.Path, eko_factory: EKOFactory):
+        """Test asserted version. Should either be supported version, or have a postrelease addition"""
+        eko = eko_factory.get()
+        eko.close()
+        eko_factory.cache = None
+        assert eko.access.path is not None
+        read_closed = EKO.read(eko.access.path, dest=tmp_path)
+        supported_versions = [13, 14, 15]
+
+        version = parse(read_closed.metadata.version)
+        assert (version.major == 0 and version.minor in supported_versions) or (
```

Comment:
let's not be so restrictive (else we need to remember to update here), so I suggest to change the first part of the `or` to `(version.major + version.minor + version.micro > 0 and not version.is_postrelease)` i.e. it is an actual release